packages/flutter/lib/src/cupertino/segmented_control.dart

```diff
@@ -147,6 +146,33 @@ class SegmentedControl<T> extends StatefulWidget {
   /// ```
   final ValueChanged<T> onValueChanged;
 
+  /// The color used to fill the backgrounds of unselected widgets and as the
+  /// text color of the selected widget.
+  ///
+  /// If this attribute is null, this color will be
+  /// [CupertinoColors.white].
+  final Color unselectedColor;
+
+  /// The color used to fill the background of the selected widget and as the text
+  /// color of unselected widgets.
+  ///
+  /// If this attribute is null, this color will be
+  /// [CupertinoColors.activeBlue].
+  final Color selectedColor;
+
+  /// The color used as the border around each widget.
+  ///
+  /// If this attribute is null, this color will be
+  /// [CupertinoColors.activeBlue].
+  final Color borderColor;
+
+  /// The color used to fill the background of the widget the user is
+  /// temporarily interacting with through a long press or drag.
+  ///
+  /// If this attribute is null, this color will be Color(0x33007aff), a light,
```

Comment:
use backticks around Color(0x33007aff)?

Reply:
Just curious, what do the backticks indicate?

Reply:
They'll just format that piece of text like code on http://docs.flutter.io. Similarly to how github does it: Color(0x33007aff) vs `Color(0x33007aff)`.